docs/governance.rst

```diff
@@ -0,0 +1,128 @@
+Binder Project Governance
+=========================
+
+The following sections describe governance and high-level principles
+surrounding the Binder Project.
+
+The Binder Project is a member of Project Jupyter, which is a fiscally
+sponsored project of NumFocus, a US 501c3 non-profit.
+
+Project mission
+~~~~~~~~~~~~~~~
+
+The mission of the Binder Project is to create, advance, and promote
+open technology that makes it easy for people to connect their data
+science communication, education, and scientific reproducibility work
+with computational environments which can run their work.
+
+The Binder Project pursues this mission by advancing the tools
+surrounding BinderHub, an open-source tool for hosting ad-hoc
+environments in the cloud. The project also serves as the primary
+maintainer of ``mybinder.org``, a public service and demonstration of
+the BinderHub technology.
+
+Membership
+~~~~~~~~~~
+
+Membership in the Binder Project is open to anyone who is nominated by
+an existing member. Members form the trusted group of individuals who
+manage the Binder Project.
+
+Membership comes with `privileges`_ and `responsibilities`_.
+
+Currently, the Binder Project team overlaps 100% with `the BinderHub
```

Comment:
Shouldn't all members of the current BinderHub team be added to the list of names?

Reply:
@betatim I slightly edited your top-level comment to **bold** the sentence where you ask people to add themselves via suggestions, I hope that's ok. I missed it too :-P